server/utils/mail_config.py

```
@@ -87,3 +96,38 @@ def send_email(to_email: str, subject: str, body: str, is_html=False):
+            "Authorization": f"Bearer {FLASK_RESEND_API_KEY}",
+            "Content-Type": "application/json",
+        }
+
+        payload = {
+            "from": f"Ecovibe Kenya <{FLASK_SMTP_USER}>",
+            "to": [to_email],
+            "subject": subject,
+            "html": body if is_html else f"<pre>{body}</pre>",
+        }
+
+        response = requests.post(
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)